### PR TITLE
Ci travis

### PR DESCRIPTION
**Pull request by Randl**

### CODE REVIEW

**Comment by crackedmind** [comment]
:white_check_mark: [Build kepka 1.0.34-aolrfjfb completed](https://ci.appveyor.com/project/procxx/tdesktop/build/1.0.34-aolrfjfb) (commit https://github.com/procxx/kepka/commit/3e47b59057 by @berkus)

**Comment by crackedmind** [comment]
:white_check_mark: [Build kepka 1.0.34-weotbeil completed](https://ci.appveyor.com/project/procxx/tdesktop/build/1.0.34-weotbeil) (commit https://github.com/procxx/kepka/commit/27c8c35487 by @Randl)

**Comment by crackedmind** [comment]
:white_check_mark: [Build kepka 1.0.34-jbpiphvn completed](https://ci.appveyor.com/project/procxx/tdesktop/build/1.0.34-jbpiphvn) (commit https://github.com/procxx/kepka/commit/48576e53a9 by @Randl)

**Comment by crackedmind** [comment]
:white_check_mark: [Build kepka 1.0.34-emwueayy completed](https://ci.appveyor.com/project/procxx/tdesktop/build/1.0.34-emwueayy) (commit https://github.com/procxx/kepka/commit/e20ada3cbc by @Randl)

**Comment by crackedmind** [comment]
:white_check_mark: [Build kepka 1.0.34-crdwqlpf completed](https://ci.appveyor.com/project/procxx/tdesktop/build/1.0.34-crdwqlpf) (commit https://github.com/procxx/kepka/commit/975148e04e by @Randl)

**Comment by crackedmind** [comment]
:white_check_mark: [Build kepka 1.0.34-tmxsfybg completed](https://ci.appveyor.com/project/procxx/tdesktop/build/1.0.34-tmxsfybg) (commit https://github.com/procxx/kepka/commit/34173d0fbf by @Randl)

**Comment by crackedmind** [comment]
:white_check_mark: [Build kepka 1.0.34-rnwrlhhu completed](https://ci.appveyor.com/project/procxx/tdesktop/build/1.0.34-rnwrlhhu) (commit https://github.com/procxx/kepka/commit/d5e47a6866 by @Randl)

**Comment by crackedmind** [comment]
:white_check_mark: [Build kepka 1.0.34-emvrevse completed](https://ci.appveyor.com/project/procxx/tdesktop/build/1.0.34-emvrevse) (commit https://github.com/procxx/kepka/commit/497d927fec by @Randl)

**Comment by crackedmind** [comment]
:white_check_mark: [Build kepka 1.0.34-uekodhee completed](https://ci.appveyor.com/project/procxx/tdesktop/build/1.0.34-uekodhee) (commit https://github.com/procxx/kepka/commit/cb025d6645 by @Randl)

**Comment by crackedmind** [comment]
:white_check_mark: [Build kepka 1.0.34-gikmfmoi completed](https://ci.appveyor.com/project/procxx/tdesktop/build/1.0.34-gikmfmoi) (commit https://github.com/procxx/kepka/commit/c16ceec8b5 by @Randl)

**Comment by crackedmind** [comment]
:white_check_mark: [Build kepka 1.0.34-ttbkvjwg completed](https://ci.appveyor.com/project/procxx/tdesktop/build/1.0.34-ttbkvjwg) (commit https://github.com/procxx/kepka/commit/11bc0a0632 by @Randl)

**Comment by crackedmind** [comment]
:white_check_mark: [Build kepka 1.0.34-voucnigg completed](https://ci.appveyor.com/project/procxx/tdesktop/build/1.0.34-voucnigg) (commit https://github.com/procxx/kepka/commit/2c0c9055e5 by @Randl)

**Comment by crackedmind** [comment]
:white_check_mark: [Build kepka 1.0.34-ehqmpgoh completed](https://ci.appveyor.com/project/procxx/tdesktop/build/1.0.34-ehqmpgoh) (commit https://github.com/procxx/kepka/commit/358f995cf2 by @Randl)

File: Telegram/ThirdParty/variant/.gitmodules [hunk 1]
```diff
@@ -1,3 +0,0 @@
-[submodule ".mason"]
```
Comment (berkus) [inline Telegram/ThirdParty/variant/.gitmodules:1]: Removed this globally

✅

**Comment by berkus** [comment]
See #47 for dockerized build on linux.

OSX upcoming.

**Pull request closed without merging**